azurerm/internal/services/compute/shared_image_version_data_source.go

```diff
@@ -133,6 +138,12 @@ func dataSourceArmSharedImageVersionRead(d *schema.ResourceData, meta interface{
 				osDiskSnapShotID = *profile.OsDiskImage.Source.ID
 			}
 			d.Set("os_disk_snapshot_id", osDiskSnapShotID)
+
+			osDiskImageSize := -1
```

Comment:
can we default this to 0, rather than -1, to match the rest of the provider here?